eternalcore-core/src/main/java/com/eternalcode/core/feature/troll/lightning/LightningCommand.java

```diff
@@ -0,0 +1,35 @@
+package com.eternalcode.core.feature.troll.lightning;
```

Comment:
Nie grupujmy tego do "troll" w ogóle raczej nie preferujemy schodkowania packages, gdzie feature to już wyjątek